### ISSUE TITLE
Xpra Maximized windows change dimension as soon as focus change

### ISSUE THREAD

**Issue by alscaldas** [Xpra-org/xpra#4112]
**Describe the bug**
When switching from a xpra window (in fact, when doing anything that changes the focus from a xpra window) if it's initially maximized it automatically resizes to a not maximized dimension. The windows never keeps maximized. If it has any other dimension (not maximized) it works correctly and keeps its dimensions.

**To Reproduce**
1. server command: /usr/bin/xpra --no-daemon start :7 (I'm using [a systemd based solution](https://wiki.archlinux.org/title/Xpra) I found in the Arch Wiki).
2. client command: xpra attach ssh://ip.of.the.server/7
3. specific action to trigger the bug: 

**System Information (please complete the following information):**
 - Server OS: Arch Linux
 - Client OS: Arch Linux
 - Xpra Server Version: xpra v6.0-r35035 (g11cca3625) beta (from [the AUR](https://aur.archlinux.org/packages/xpra-git)).
 - Xpra Client Version: xpra v6.0-r35035 (g11cca3625) beta (also from [the AUR](https://aur.archlinux.org/packages/xpra-git)).

**Additional context**
- The Xpra server is running in a qemu kvm (not sure if it's relevant to this issue. I tried using both Spice with Virtio and QXL).
- I'm using KDE on both the client and server.
- I have the same behavior no matter if I use wayland or X11 on the client.
- Before opening this issue I looked around here and found [this](https://github.com/Xpra-org/xpra/issues/2423), that seems to describe a very similar situation to mine, but as it's already closed I'm not sure if it has the same origin.

Based on the "reporting bugs" in the wiki section, I'm attaching the following:

[server_xpra_showconfig.txt](https://github.com/Xpra-org/xpra/files/14099412/server_xpra_showconfig.txt)
[client_xpra_showconfig.txt](https://github.com/Xpra-org/xpra/files/14099444/client_xpra_showconfig.txt)
[server_xpra_info.txt](https://github.com/Xpra-org/xpra/files/14099696/server_xpra_info.txt)

- *is this a new problem or a regression? if this is a regression, when did it start?* I don't know, since I've been using xpra for no more than a week.
- *are there other bugs that are similar?* Well, I believe [this](https://github.com/Xpra-org/xpra/issues/2423) may be related.
- *are there other unusual issues / behaviour apart from this particular bug?* No.
- *is it reproducible reliably? and if so, how?* Yes. If I maximize a xpra window, as soon as a change focus (alt + tab, for example, or even when I open another xpra window based of the first one, as for example open Dolphin and click on a video or image, which opens gwenview or mpv).
- does this happen with all picture encodings? Sorry... I don't know what are picture encodings.
- does switching off certain features make a difference (ie: sound, opengl, clipboard). Not to the extent of my tests...
- does re-connecting help? No.
- does it happen with other type of clients (different OS, etc). I don't know. The only SO I have available to test is a Linux client. But I don't believe I could use it seamless in a Windows client, so I don't believe it'd be a valid test.

```
[myuser@myclient~]$ echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/bin:/home/myuser/.local/share/flatpak/exports/bin:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/mysuser/.local/bin:/home/myuser/.cargo/bin:/home/myuser/.local/share/gem/ruby/3.0.0/bin:/home/myuser/.local/bin:/home/myuser/.cargo/bin:/home/myuser/.local/share/gem/ruby/3.0.0/bin
```

```
[myuser@myclient ~]$ echo $LD_LIBRARY_PATH

```

```
[myuser@myserver ~]$ echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/myuser/.local/bin:/home/myuser/.local/bin:/home/myuser/.local/bin:/home/myuser/.local/bin
```

```
[myuser@myserver ~]$ echo $LD_LIBRARY_PATH

```

**Comment by totaam** [comment]
Sounds like https://github.com/Xpra-org/xpra/pull/4108
Does that change fix the problem?

**Comment by alscaldas** [comment]
Hey @totaam, thanks for taking the time to look into this issue. If I read it right and the referred changes were commited 3 days ago then I believe it didn't work for me.

I'm now using `xpra v6.0-r35071 (gf12369ad4) beta` (which is posterior to the referred commit) and still have this issue.

**Comment by totaam** [comment]
@alscaldas I haven't had a chance to look at these changes yet, they're in a PR and not merged yet.
(and I'm not convinced this is the correct way of fixing this issue either - just want to confirm that it is indeed the same issue)

**Comment by alscaldas** [comment]
@totaam I see. That said, since they aren't merged and installing through AUR (yay) probably won't work what am I supposed to do to test them, so that I might provide some additional (and probably useful) info?

**Comment by totaam** [comment]
You can manually edit that file and add the two non-comment lines from:
https://github.com/Xpra-org/xpra/pull/4108/files

**Comment by totaam** [comment]
@alscaldas never mind, I've tested it and fixed the bug in the commit above.
This will be included in v5.0.5 too.
